bridgestyle/qgis/togeostyler.py

```diff
@@ -345,6 +345,19 @@ def processLabeling(layer, labeling, name="labeling", filter=None):
     if filter is not None:
         result["filter"] = filter
 
+    # labels rendering scale-based visibility
+    if labeling is not None:
+        settings = layer.labeling().settings()
+        sv = settings.scaleVisibility
+        if sv:
+            scale = {"max": settings.minimumScale, "min": settings.maximumScale}
+            result["scaleDenominator"] = scale
+
+    # layer rendering scale-based visibility
+    if (layer.hasScaleBasedVisibility()):
+        scale = processRuleScale(layer)
+        result["scaleDenominator"] = scale
+
     if hasattr(labeling, 'dependsOnScale') and labeling.dependsOnScale():
         scale = processRuleScale(labeling)
         result["scaleDenominator"] = scale
```

Comment:
Hi @geraldo, good addition, but I would rewrite this like so:

```suggestion
    # labels rendering scale-based visibility
    scale = None
    if settings.scaleVisibility:
        scale = processRuleScale(settings.scaleVisibility)
    elif hasattr(labeling, 'dependsOnScale') and labeling.dependsOnScale():
        scale = processRuleScale(labeling)
    elif layer.hasScaleBasedVisibility():
        scale = processRuleScale(layer)
        
    if scale:
        result["scaleDenominator"] = scale
```

Note that the `settings` object has already been set at the top of the function. That would also fail if `labeling` is `None`, so that check is not required either.

Please provide feedback on my PR #87. Then I'll merge that first and then you can update this PR. After this one is merged, I want to do a new QGIS Bridge release.